test/reserveSetup.js

```diff
@@ -0,0 +1,195 @@
+const ConversionRates = artifacts.require("MockEnhancedStepFunctions.sol");
+const KyberFprReserveV2 = artifacts.require("KyberFprReserveV2");
+
+const Helper = require("./helper.js");
+const BN = web3.utils.BN;
+
+//global variables
+//////////////////
+const {precisionUnits, ethAddress, zeroAddress, 
+    zeroBN} = require("./helper.js");
+
+//block data
+let validRateDurationInBlocks = 1000;
+let maxGasPrice = new BN(150).mul(new BN(10).pow(new BN(9))); // 150 * 10^9
+
+// imbalance data
+let minimalRecordResolution = new BN(2);
+let maxPerBlockImbalance = 40000;
+let maxTotalImbalance = maxPerBlockImbalance * 12;
+
+
+//imbalance buy steps
```

Comment:
lets update this to a more traditional setup
so buy imbalance x values are only positive
and sell imbalance x values are negative.

also nice to add the option to pass in steps input

Reply:
Ok, for steps input can just modify this initial array data

Reply:
OK